tests/js/e2e-tests-helpers.js

```diff
@@ -0,0 +1,70 @@
+import {
+  getAllBlocks,
+  selectBlockByClientId,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Selects a block by name.
+ *
+ * @param {string} name The block name.
+ */
+export const selectBlockByName = async ( name ) => {
+  await selectBlockByClientId(
+    ( await getAllBlocks() ).find( ( block ) => block.name === name ).clientId
```

Comment:
Don't forget to catch the case where the block is not found.

Reply:
I guess it is expected that this throws an exception, but catching it would still allow to throw a more meaningful error.

Reply:
Added a `throw` in case the block is not found. Tested it and it fails gracefully.  👍